dojo/tools/spotbugs/parser.py

```diff
@@ -43,6 +43,15 @@ def __init__(self, filename, test):
             impact = 'N/A'
             references = 'N/A'
 
+            # find the source line and file on the buginstance
+            source_line = "N/A"
```

Comment:
`line` in the model is an integer. If this never gets set anywhere else, you then have a type mismatch.

Reply:
but, this work on my version actually....
Will correct it